<launch>

  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" output="screen"/>

  <node pkg="homework2" type="h2_service_node.py" name="h2_service_node" output="screen">
    <param name="last_movement_type" value="initialized" type="string"/>
  </node>

  <node pkg="homework2" type="h2_client_node.py" name="h2_client_node" output="screen"/>

</launch>